docs/integrations/builtin/core-nodes/n8n-nodes-base.httprequest/common-issues.md

```diff
@@ -0,0 +1,60 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: HTTP Request node common issues 
+description: Documentation for common issues and questions in the HTTP Request node in n8n, a workflow automation platform. Includes details of the issue and suggested resolutions.
+contentType: integration
+priority: critical
+---
+
+# HTTP Request node common issues
+
+Here are some common errors and issues with the [HTTP Request node](/integrations/builtin/app-nodes/n8n-nodes-langchain.openai/) and steps to resolve or troubleshoot them.
+
+## Bad request - please check your parameters
+
+This error displays when the node receives a 400 error indicating a bad request. This error most often occurs because:
+
+* You're using an invalid name or value in a **Query Parameter**.
+* You're passing array values in a **Query Parameter** but the array isn't formatted correctly. Try using the [**Array Format in Query Parameters**](/integrations/builtin/core-nodes/n8n-nodes-base.httprequest/#array-format-in-query-parameters) option.
+
+Review the API documentation for your service to format your query parameters.
+
+<!-- vale off -->
+## The resource you are requesting could not be found
+<!-- vale on -->
+
+This error displays when the endpoint **URL** you entered is invalid.
+
+This may be due to a typo in the URL or a deprecated API. Refer to your service's API documentation to verify you have a valid endpoint.
+
+## JSON parameter need to be an valid JSON
```

Comment:
This looks like it's probably a typo, but I don't know what the actual, valid error string is to correct it.

Reply:
@imchairmanm this is the actual text as it exists in n8n. Yes, of course it's a typo, but I also want people searching the error to actually turn up this document, so I had to mimic the typo as it existed in the UI. Aya and I've been tracking typos in the UI [here](https://linear.app/n8n/issue/DOC-1003/ui-fixes-typos-noticed-while-working-on-the-docs-overhaul) and I added that one. :)

Reply:
Perfect. Thank you!